content/2016-04-05-benchmarks.rst

```diff
@@ -0,0 +1,165 @@
+Compiler Flags
+##############
+
+:date: 2016-04-05
+:category: benchmark
+:lang: en
+:authors: Lightjohn
+:summary: Let's re-run an existing benchmark with pythran 
+
+The benchmark
+=============
+
+First there was a paper [0]_, 
+in the paper there was a github [1]_, 
+in the github [1]_ some benchmarks. 
+In my case I wanted to re-run the julia code because the language is changing quickly and so may do better now. 
+But the day before I discovered `Pythran` so why not test both?
+
+And so let's re-run two benchmarks: `C++ <https://github.com/jesusfv/Comparison-Programming-Languages-Economics/blob/master/RBC_CPP.cpp>`_ , `Julia <https://github.com/jesusfv/Comparison-Programming-Languages-Economics/blob/master/RBC_Julia.jl>`_ and add a new one, Pythran.
+
+C++ results
+-----------
+
+Compiling and running C++ code was easy:
+
+.. code:: sh
+
+	% g++ -Ofast RBC_CPP.cpp -o testcpp
```

Comment:
to be fair, you should use ``-O3`` because ``-Ofast`` triggers some non-conservative optimizations.